src/relo.coffee

```diff
@@ -16,13 +16,17 @@ spawn = ->
   program = options.command[0]
   argv    = options.command[1..]
 
+  if ! argv.length && program.search(' ') != -1
+    [program, argv...] = program.split(' ')
+
   subproc = cp.spawn program, argv,
     cwd     : process.cwd()
-    stdio   : 'inherit'
     detached: true
 
   debug "Started #{program}"
 
+  subproc.stdout.on 'data', (buffer) -> console.log(buffer.toString().trim())
```

Comment:
Hello @NicoSantangelo! Thanks for this contribution :D! Why do you `.trim()` here?